src/components/MyAccount/Settings/AccountSettingsUtils.ts

```diff
@@ -97,23 +101,31 @@ export const parseAccountSettingsPayload = (
           },
         }
         break
-      case "homeLibraryCode":
-        putRequestPayload.homeLibraryCode = fieldValue
+      case "homeLibrary":
+        putRequestPayload.homeLibraryCode = fieldValue.split("@")[0]
```

Comment:
Should this be `putRequestPayload.homeLibrary`?

Reply:
I think I answered my own question by looking more into this PR, but I'm a bit confused where the `homeLibraryCode` value is used.

Reply:
`homeLibraryCode` is only used for put request bodies